frontend/model/contracts/group.js

```diff
@@ -788,6 +839,63 @@ sbp('chelonia/defineContract', {
         })
       }
     },
+    'gi.contracts/group/addChatRoom': {
+      validate: objectMaybeOf({
+        chatRoomID: string,
+        attributes: objectMaybeOf({
+          name: string,
+          type: unionOf(...Object.values(CHATROOM_TYPES).map(v => literalOf(v))),
+          privacyLevel: unionOf(...Object.values(CHATROOM_PRIVACY_LEVEL).map(v => literalOf(v)))
+        })
```

Comment:
Having these `attributes` here is a DRY violation, as we already have these exact same attributes stored within the chatroom contract itself. Having to maintain them in two different places creates extra code that has to be maintained and creates potentials for bugs if any of these values get out of sync from one another.

Is it possible to make it so that we don't have to pass these duplicates in here?

Reply:
Understood. Good point. I will fix that DRY violation.